apps/dashboard/src/components/workflow-editor/in-app-preview.tsx

```diff
@@ -29,7 +29,7 @@ export const InAppPreview = (props: InAppPreviewProps) => {
   return (
     <div
       className={cn(
-        'border-foreground-200 to-background/90 pointer-events-none relative mx-auto flex h-full w-full flex-col gap-4 rounded-xl px-2 py-3 shadow-sm',
+        'border-foreground-200 to-background/90 pointer-events-none relative mx-auto flex h-full w-full flex-col gap-4 rounded-xl px-4 py-3 shadow-sm',
```

Comment:
Fix alignment issues between the body and the preview header. Padding and margin should always be on the layout level, not on each component of the preview.